homeassistant/components/ephember/climate.py

```diff
@@ -89,8 +89,9 @@ def __init__(self, ember, zone):
         self._ember = ember
         self._zone_name = zone_name(zone)
         self._zone = zone
-        self._hot_water = zone_is_hot_water(zone)
-
+        self._hot_water = zone['deviceType'] == 4
+        """4 is a specific device type for immersions returned by EPH. Hot Water temp cannot be changed"""
+        
```

Comment:
```suggestion
        # 4 is a specific device type for immersions returned by EPH. Hot Water temp cannot be changed
        self._hot_water = zone['deviceType'] == 4       
```

Reply:
Confirmed that the pypi 0.3.1 release is 6ed0c14d3474bead9d0922021a65f8b5cce72fb7 and 0.4.0 matches b5226cc582c9805aee938dc4bfd4a4a28915b21b. So the compare link above is accurate.